src/web/mjs/connectors/MangaIndoWeb.mjs

```diff
@@ -1,77 +1,63 @@
 import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
 
-/**
- *
- */
 export default class MangaIndoWeb extends Connector {
 
-    /**
-     *
-     */
     constructor() {
         super();
         super.id = 'mangaindoweb';
         super.label = 'MangaIndoWeb';
         this.tags = [ 'manga', 'indonesian' ];
         this.url = 'https://mangaindo.web.id';
+        this.path = '/manga-list/';
+
+        this.queryMangas = 'div#az-slider div.letter-section ul li.manga-list a';
+        this.queryChapters = 'ul.lcp_catlist li a';
+        this.queryPages = 'div.entry-content source[src]:not([src=""])';
+        this.querMangaTitleFromURI = 'div#main article div.title h2';
+    }
+
+    async _getMangas(){
+        const URI = new URL(this.path, this.url);
+        const request = new Request(URI, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangas);
+
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink( element, request.url ),
```

Comment:
Use `this.url` instead of `request.url` for consistency, as `this.url` is also used in `_getChapters` when composing the URL agaion